Dockerfile

```diff
@@ -150,6 +151,50 @@ RUN mkdir -p /vboxguest && \
     mkdir -p $ROOTFS/sbin && \
     cp x86/lib/VBoxGuestAdditions/mount.vboxsf $ROOTFS/sbin/
 
+# Build VMware Tools
+ENV OVT_VERSION 9.4.6-1770165
+
+# Download and prepare ovt source
+RUN mkdir -p /vmtoolsd && cd /vmtoolsd && \
+    curl -L -o open-vm-tools-$OVT_VERSION.tar.gz http://downloads.sourceforge.net/open-vm-tools/open-vm-tools-$OVT_VERSION.tar.gz && \
+    tar xfz open-vm-tools-$OVT_VERSION.tar.gz && rm open-vm-tools-$OVT_VERSION.tar.gz &&\
+    mv open-vm-tools-$OVT_VERSION open-vm-tools
```

Comment:
Why not use `--strip-components` on `tar` to avoid the need to move the directory afterwards?  Something like:
```Dockerfile
# Download and prepare ovt source
RUN mkdir -p /vmtoolsd/open-vm-tools \
    && curl -L http://downloads.sourceforge.net/open-vm-tools/open-vm-tools-$OVT_VERSION.tar.gz \
        | tar -xzC /vmtoolsd/open-vm-tools --strip-components 1
```

That way, we can even avoid the on-disk tar file completely.